pkg/kubectl/cmd/get.go

```diff
@@ -203,3 +217,17 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return printer.PrintObj(r.Object, out)
 	})
 }
+
+func filterNonRunningPodsSelector() (fields.Selector, error) {
+	// TODO: use correct fields selector for pod list
+	field := "status.phase!=Succeeded,status.phase!=Failed"
```

Comment:
Perheps you need to add status.phase to the fieldmapping here: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/client/request.go#L314